Makefile

```diff
@@ -328,6 +328,7 @@ ifeq ($(SYSTEM),Darwin)
 CXXFLAGS += -stdlib=libc++
 endif
 CPPFLAGS += -g -Wall -Wextra -Werror -Wno-long-long -Wno-unused-parameter -DOSATOMIC_USE_INLINED=1 -Ithird_party/abseil-cpp
+COREFLAGS += -fno-rtti -fno-exceptions
```

Comment:
is it safe to use fno-exceptions?

Reply:
It should be for now since this is only passed to core. If we ever want to start statically linking and using libstd, then we would need to reconsider. But I'd rather have this for now, then whoever needs exceptions in core first can remove it, if they have a convincing argument